packages/webapp/src/routes/land/upload-legal-agreement.js

```diff
@@ -139,11 +155,14 @@ export default [{
 }
 ]
 
-const maximumFileSizeExceeded = h => {
-  return getMaximumFileSizeExceededView({
-    h,
-    href: legalAgreementId,
-    maximumFileSize: process.env.MAX_GEOSPATIAL_LAND_BOUNDARY_UPLOAD_MB,
-    view: constants.views.UPLOAD_LEGAL_AGREEMENT
+const maximumFileSizeExceeded = (h, legalAgreementType) => {
+  return h.view(constants.views.UPLOAD_LEGAL_AGREEMENT, {
```

Comment:
Rather than remove reference to getMaximumFileSizeExceededView couldn't the function be updated with an optional model data, legalAgreementType in this case?

Reply:
can we consider this as a technical dept a its a very common function , need to be done carefully

Reply:
That's fine